src/agents/triggers/trigger.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * All triggers are invisible entities that cause the game engine to
+ * act in an unique way.
+ */
+function Trigger(game, x, y, width, height) {
```

Comment:
Makes sense. I'll make this change on one of the commit states based on which one we decide to go with.